content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst

```diff
@@ -52,64 +52,46 @@ quotation form. On the blank quotation form, select a :guilabel:`Customer`, add
+:guilabel:`Inventory Overview` dashboard.
 
-.. image:: delivery_three_steps/three-step-delivery-transfers.png
-   :align: center
-   :alt: Ready status for the pick operation while the packing and delivery operations are waiting
-         another operation.
-
-The status of the picking will be :guilabel:`Ready`, since the product must be picked from stock
-before it can be packed. The status of the packing order and delivery order will both be
-:guilabel:`Waiting Another Operation`, since the packing and delivery cannot happen until the
-picking is completed. The status of the delivery order will only change to :guilabel:`Ready` when
-the packing has been marked :guilabel:`Done`.
-
-The receipt can also be found in the *Inventory* application. In the :guilabel:`Overview` dashboard,
-click the :guilabel:`1 To Process` smart button in the :guilabel:`Pick` kanban card.
-
-.. image:: delivery_three_steps/three-step-kanban-pick.png
-   :align: center
-   :alt: The pick order can be seen in the Inventory Kanban view.
+Click the :guilabel:`# To Process` button, which reveals the picking order generated from the
+previously confirmed sales order.
 
 Click on the picking to process. If the product is in stock, Odoo will automatically reserve the
 product. Click :guilabel:`Validate` to mark the picking as done, and complete the transfer to the
-:guilabel:`Packing Zone`. Then the packing order will be ready. Since the documents are linked, the
-products which have been previously picked are automatically reserved on the packing order.
+:guilabel:`Packing Zone`.
 
-.. image:: delivery_three_steps/validate-three-step-pick.png
+.. image:: delivery_three_steps/delivery-three-steps-picking-order.png
    :align: center
-   :alt: Validate the picking by clicking Validate.
+   :alt: Picking order operation showing source location and destination location.
 
 Process a packing
 -----------------
 
-The packing order will be ready to be processed once the picking is completed, and can be found in
-the :menuselection:`Inventory` application :guilabel:`Overview` dashboard. Click the :guilabel:`1 To
-Process` smart button in the :guilabel:`Pack` kanban card.
+After validating the picking, the packing order is ready to process. Click back to the
+:guilabel:`Inventory Overview`, and locate the :guilabel:`Pack` task card on the dashboard.
 
-.. image:: delivery_three_steps/three-step-kanban-pack.png
-   :align: center
-   :alt: The packing order can be seen in the Inventory kanban view.
+Click the :guilabel:`# To Process` button (in this case, :guilabel:`1 To Process`). This reveals the
+packing order generated from the previously confirmed sales order.
 
 Click on the packing order associated with the sales order, then click on :guilabel:`Validate` to
 complete the packing.
 
-.. image:: delivery_three_steps/validate-three-step-pack.png
+.. image:: delivery_three_steps/delivery-three-steps-packing-order.png
```

Comment:
Same here...zooming in like this isn't very helpful I don't think since we're removing surrounding UI elements that give the reader navigational context.

As well, if we're going to take the server hit for storing the screenshot, I'd argue that ideally we super fill these out like a living success model, so we can get the most use out of it (and so would our readers).

Reply:
Screenshot has been replaced with a new zoomed-out one, showcasing more of the navigational UI and highlighting key areas. Suggestion implemented ✅